api-reference/beta/api/applicationsecurityprofile_list.md

```diff
@@ -0,0 +1,97 @@
+# List applicationSecurityProfiles
+
+Retrieve a list of applicationSecurityProfile objects.
+
+## Permissions
+
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](../../../concepts/permissions_reference.md).
+
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) |  SecurityEvents.Read.All, SecurityEvents.ReadWrite.All  |
+|Delegated (personal Microsoft account) | Not supported   |
+|Application | SecurityEvents.Read.All, SecurityEvents.ReadWrite.All |
+
+## HTTP request
+
+<!-- { "blockType": "ignored" } -->
+
+```http
+GET /security/applicationSecurityProfiles
+GET /security/applicationSecurityProfiles?$top=1
+GET /security/applicationSecurityProfiles?$filter={property} eq '{property-value}'
+GET /security/applicationSecurityProfiles?$filter={property} eq '{property-value}'&$top=5
+GET /security/applicationSecurityProfiles?$filter={property} eq '{property-value}'&{property} eq '{property-value}'
+```
+
+## Optional query parameters
+
+This method supports the `$count, $filter, $orderby, $select, $skip, and $top` [OData Query Parameters](http://graph.microsoft.io/docs/overview/query_parameters) to help customize the response.

```

Comment:
Please make the same update to the parameters section per earlier comment. Also note that we shouldn't link to the former URL from the site; this should be a relative link to the MD file.